### PR TITLE
stolostron/backlog#20292 Minor (do not merge)

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/cluster_size.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [#sizing-your-cluster]
```
Comment (cadawson) [inline install/cluster_size.adoc:1]: I think we talked about changing this to hub cluster, if it is applicable. Based on what Randy mentioned, that might still be a good idea. 

I see where "lower bound" could be confused as a minimum requirement. Should we explain it?

Should "Gb" be "GB"?

Will there be a counterpart to "Product environment"?

**Review by cadawson — COMMENTED** [review]
I think the list format looks good for this content. I added some questions that I had, based on what Randy mentioned, and what I noticed. Thanks!

File: install/cluster_size.adoc (hunk 1 above)
Comment (swopebe) [reply install/cluster_size.adoc:1]: @cadawson you likely saw that @randymgeorge already addresses this in the issue comment I sent you, so I feel we should remain in the issue for now for efficiency. Please continue the discussion in the issue so we don't have multiple branches addressing this.
.

**Comment by swopebe** [comment]
closing, I did a lot of work in the cluster doc to clean this up: https://github.com/stolostron/backlog/issues/20292

**Comment by swopebe** [comment]
Best practice: Slack/GitHub to communicate and avoid redundant work.

**Pull request closed without merging**